redbot/cogs/economy/economy.py

```diff
@@ -536,15 +543,58 @@ async def leaderboard(self, ctx: commands.Context, top: int = 10, show_global: b
                     f"<<{author.display_name}>>\n"
                 )
             if pos % 10 == 0:
-                highscores.append(box(temp_msg, lang="md"))
+                if embed_requested:
+                    embed = base_embed.copy()
+                    embed.description = box(temp_msg, lang="md")
+                    embed.set_footer(
+                        text=_(
+                            "You are {author_pos}/{len_board}. Page {page_num}/{page_len}."
+                            " Use {left} and {right} to navigate."
+                        ).format(
+                            author_pos=bank.get_leaderboard_position(author),
```

Comment:
this is missing an await on `bank.get_leaderboard_position` however please dont do this, the this to a variable outside the loop and then reference this variable here and in L578

Reply:
Thank you for spotting that! I've now fixed this in 516cf91fc6310968848f431410764debbe4d828d :)